pom.xml

```diff
@@ -10,26 +10,26 @@
   <parent>
     <groupId>edu.tamu.weaver</groupId>
     <artifactId>webservice-parent</artifactId>
-    <version>2.0.2</version>
+    <version>2.1.1-RC8</version>
   </parent>
 
   <properties>
     <start-class>edu.tamu.app.WebServerInit</start-class>
   </properties>
 
-  <packaging>war</packaging>
+  <!-- <packaging>war</packaging> -->
```

Comment:
Please uncomment and change to `<packaging>jar</packaging>`

Reply:
Modified war to jar